vars/buildDotNetCore.groovy

```diff
@@ -51,6 +54,10 @@ def call(Map config=[:]) {
         build.runTests(repoName, repoName, BUILD_NUMBER, tag)
       }
 
+     stage('Publish pact broker') {
+        pact.pacts(repoName, pact.string, pact.usernamePassword)
```

Comment:
Can probably reduce this to just `repoName` and have the class itself encapsulate all the logic